functional/tests/core.test.mjs

```diff
@@ -0,0 +1,151 @@
+"use strict";
+
+import { expect, jest, test, afterAll, beforeAll } from "@jest/globals";
+import * as utilities from "./utilities.mjs";
+
+jest.setTimeout(20000);
+
+beforeAll(async () => {
+  console.log(await utilities.mfState(true));
+});
+
+afterAll(async () => {
+  //Stop Mock Firebolt
+  console.log(await utilities.killPort(9998));
+});
+
+test(`MF Startup/Health Check`, async () => {
```

Comment:
why do we need to run it here again?  since its already added in health-check.test.mjs

Reply:
Yes, this one with default port.

Reply:
It should be ok, I believe this can be moved to appropriate test.mjs

Reply:
As it's running with default port and with core only, I kept it here.